src/ray/gcs/store_client/in_memory_store_client.cc

```diff
@@ -132,6 +144,37 @@ std::shared_ptr<InMemoryStoreClient::InMemoryTable> InMemoryStoreClient::GetOrCr
   }
 }
 
+Status InMemoryStoreClient::AsyncGetKeys(
+    const std::string &table_name,
+    const std::string &prefix,
+    std::function<void(std::vector<std::string>)> callback) {
+  RAY_CHECK(callback);
```

Comment:
Maybe we should use if? (is caller guaranteed to have callback?)

Reply:
yeah looks like it.